tasks/build/images.js

```diff
@@ -3,15 +3,27 @@
 const gulp = require('gulp'),
   config = require('../../config'),
   changed = require('gulp-changed'),
-  cache = require('gulp-cache'),
   imagemin = require('gulp-imagemin'),
-  notify = require('gulp-notify');
+  notify = require('gulp-notify'),
+  ignore = require('gulp-ignore'),
+  filter = require('gulp-filter');
 
 module.exports = function() {
+  const allButScreenshot = filter(['**/*', '!**/screenshot.*'], {restore: true});
+  const onlyScreenshot = filter(['**/screenshot.*']);
```

Comment:
Noting for future reference:

Although the [documentation](https://codex.wordpress.org/Theme_Development#Screenshot) says that `.png` is expected, `.jpg`, `.jpeg` and `.gif` are also  allowed, though not recommended.